src/ipinfoWrapper.ts

```diff
@@ -351,11 +351,12 @@ export default class IPinfoWrapper {
     }
 
     /**
-     * Get the result of a list of URLs in bulk.
+     * Allow a batch endpoint to group together IPs and retrieve details in bulk.
      *
      * @param urls the array of URLs.
-     * @param batchSize default value is 1000.
-     * @param batchTimeout default value is 5000 milliseconds.
+     * @param batchSize default value is set to max value for batch size, which 
+     * is 1000.
```

Comment:
Think this should be on the same line as the param right? Or was it breaking something?

Reply:
Yeah, not breaking anything, was going little bit lengthy. One liner can do here though, I'll update it.